kafka-connect/s2i/resources/openshift-template.yaml

```diff
@@ -13,6 +13,10 @@ parameters:
   name: TARGET_IMAGE_NAME
   required: true
   value: "kafka-connect"
+- description: Specifies the tag of the new Kafka Connect Docker image which will be generated
```

Comment:
The "Specifies" is redundant: every description could start with "Specifies...".